spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/ListenerContainerIdleEvent.java

```diff
@@ -42,7 +43,7 @@
 
 	public ListenerContainerIdleEvent(Object source, long idleTime, @Nullable String id, String... queueNames) {
 		super(source);
-		this.idleTime = idleTime;
+		this.idleTime = Duration.ofMillis(idleTime);
```

Comment:
What made you to wrap, when you still use a simple long afterwards?

Reply:
For the `toString()` (remove the magic number 1000).